include/linker/common-rom.ld

```diff
@@ -36,17 +36,6 @@
 	}
 #endif
 
-#ifdef CONFIG_COVERAGE_HW_TEST
```

Comment:
you added this in previous commit, now moving it, can all of this be squashed in one single commit?